### ISSUE TITLE
Implement three column layout

### ISSUE THREAD

**Issue by j0hj0h** [hbz/oerworldmap#577]
- Overall app behaviour of layout initialization with three columns: map, aggregation, resource
- UI elements to open and collapse columns
- Three modes for aggregation column: flying, list, statistics
- Switch list to endless scroll

**Comment by j0hj0h** [comment]
Drafts for three column layout, new filtersections and modals by @formimkontext
![20160228-oer-world-map-01-landingpage](https://cloud.githubusercontent.com/assets/3071485/13984450/996047ea-f0f6-11e5-9a51-7800fd8b026c.jpg)
![20160228-oer-world-map-02-landingpage](https://cloud.githubusercontent.com/assets/3071485/13984452/99609c9a-f0f6-11e5-8c97-6d6d1844cdcd.jpg)
![20160228-oer-world-map-03-aggregation](https://cloud.githubusercontent.com/assets/3071485/13984451/99607b70-f0f6-11e5-9903-9f2b801eae07.jpg)
![20160228-oer-world-map-04-aggregation](https://cloud.githubusercontent.com/assets/3071485/13984453/9960aea6-f0f6-11e5-95af-419e301d4909.jpg)
![20160228-oer-world-map-05-aggregation-filter-type](https://cloud.githubusercontent.com/assets/3071485/13984454/9961c5f2-f0f6-11e5-8ef7-d72630e5c3ec.jpg)
![20160228-oer-world-map-06-aggregation-filter-dropdown](https://cloud.githubusercontent.com/assets/3071485/13984455/99623154-f0f6-11e5-82f4-caf6a3d50939.jpg)
![20160228-oer-world-map-07-aggregation-filter-aktiv](https://cloud.githubusercontent.com/assets/3071485/13984456/9977dd56-f0f6-11e5-8901-c7dddc19eda7.jpg)
![20160228-oer-world-map-08-aggregation-filter-event](https://cloud.githubusercontent.com/assets/3071485/13984457/99788a44-f0f6-11e5-8850-d6b7e5184fa6.jpg)
![20160228-oer-world-map-09-aggregation-country-champion](https://cloud.githubusercontent.com/assets/3071485/13984460/99799998-f0f6-11e5-87b1-dfbdb8a1e757.jpg)
![20160228-oer-world-map-10-aggregation resource-1](https://cloud.githubusercontent.com/assets/3071485/13984458/9978e0c0-f0f6-11e5-8a94-3ea153c84c07.jpg)
![20160228-oer-world-map-11-aggregation resource-2](https://cloud.githubusercontent.com/assets/3071485/13984459/997943e4-f0f6-11e5-83aa-63bd85f1a927.jpg)
![20160228-oer-world-map-12-aggregation resource-3](https://cloud.githubusercontent.com/assets/3071485/13984461/997a34de-f0f6-11e5-8cf5-2d406ed9e93d.jpg)

**Comment by j0hj0h** [comment]
After implementing the three column layout the modals are not working anymore, because we restructured the markup. Up to now, the modals worked with nothing but native bootstrap data-* attributes. The entire markup for each modal was loaded with each navigation step in the `<body>`s "root-level", where it stayed and could be targeted directly.

The main reason why this doesn't work any longer:
The new client routes load content directly into columns, but bootstrap modals should be in a "top-level position in your document" (see http://getbootstrap.com/javascript/#modals)

Beside from that we tided up the markup, removing all the modal containers and assigned the IDs (that are needed to target the modals) directly to the forms. Ok, this is reversable, but as a modal needs four nested `<div>`s ... it would be great to leave that out.

So there are three questions:
- How to wrap the modal markup arround?
- How to target them?
- How do they get in the `<body>`s "root-level"?

Two possible solutions:
- Implement custom modal handling, that wraps the modal markup around content, places it in the `<body>`-root and opens the modal on the fly, leaving a marker at the old place where it is send back to, when the modal closes.
- Wrap and move the modal contents to the `<body>`-root on attach and remove them when the content where they come from is replaced (which automatically happens in the first option). This way we could stick to the native data-* attribute technique.

The first option looks simpler for me so far. I'll try it ...
